lib/shrine/plugins/versions.rb

```diff
@@ -105,13 +105,7 @@ def url(version = nil, **options)
 
           if attachment.is_a?(Hash)
             if version
-              if attachment.key?(version)
```

Comment:
This solution seems a bit complex, wouldn't just adding

```rb
version = version.to_sym
```

above this line work?

Reply:
@janko yup, it will work indeed. Though my solution won't break anything in case any existing version name is for any reason not a Symbol.
If you feel like that could never happen - then your simpler solution fits better :)

Reply:
@janko You see, we're only symbolizing version names in `Shrine.uploaded_file`, which converts a hash of data into a hash of versions, mostly during the read from DB I believe.

But what if that hash of versions was somehow modified afterward?
Can we guarantee all the keys are going to be symbols? :thinking: 

I'm just a bit paranoic you see...

Reply:
Anyway, I didn't want to introduce potential bugs with this my little enhancement.

Reply:
Thanks for being thorough, and thanks for the PR in general! I think it would be very strange if a Shrine user intercepted reading from the DB and added some versions with string keys, so I believe the approach I suggested shouldn't cause any problems.